Modules/IO/NIFTI/src/itkNiftiImageIO.cxx

```diff
@@ -2151,6 +2151,24 @@ NiftiImageIO::SetNIfTIOrientationFromImageIO(unsigned short origdims, unsigned s
   mat44 matrix =
     nifti_make_orthog_mat44(dirx[0], dirx[1], dirx[2], diry[0], diry[1], diry[2], dirz[0], dirz[1], dirz[2]);
   matrix = mat44_transpose(matrix);
+  // Check if matrix is orthogonal and issue a warning if it was
+  // coerced to orthogonal. Use same epsilon as used in SetImageIOOrientationFromNIfTI
+  const unsigned int matdim(this->GetDirection(0).size());
+  vnl_matrix<float>  imagemat(matdim, matdim);
+  for (unsigned c = 0; c < matdim; ++c)
+  {
+    auto col = this->GetDirection(c);
+    for (unsigned r = 0; r < matdim; ++r)
+    {
+      imagemat[r][c] = col[r];
+    }
+  }
+  auto candidate_identity = imagemat * imagemat.transpose();
```

Comment:
Underscores are strongly discouraged in C++ as per the ITK style guidelines: https://itk.org/ItkSoftwareGuide.pdf (section C.6)

Reply:
Prefer to keep this as it is consistent with the naming in `SetImageIOOrientationFromNIfTI` .